global.json

```diff
@@ -1,3 +1,6 @@
 {
-    "projects": ["src"]
+    "projects": [ "src" ],
+    "sdk": {
+        "version": "1.0.0-beta8"
```

Comment:
I don't think this modification should be part of the PR. Especially not since the latest stable version is RC1, not beta8.